apps/web/.env.example

```
@@ -141,4 +141,5 @@ LOOPS_API_SECRET=
 # Feature flags
 # NEXT_PUBLIC_DIGEST_ENABLED=true
 # NEXT_PUBLIC_MEETING_BRIEFS_ENABLED=true
-# NEXT_PUBLIC_INTEGRATIONS_ENABLED=true
\ No newline at end of file
+# NEXT_PUBLIC_INTEGRATIONS_ENABLED=true
+# NEXT_PUBLIC_COMMAND_PALETTE_ENABLED=true
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)